pkg/trait/mount_test.go

```diff
@@ -160,9 +168,10 @@ func getNominalEnv(t *testing.T, traitCatalog *Catalog) *Environment {
 				},
 				Traits: v1.Traits{
 					Mount: &traitv1.MountTrait{
-						Configs:   []string{"configmap:my-cm"},
-						Resources: []string{"secret:my-secret"},
-						Volumes:   []string{"my-pvc:/over/the/rainbow"},
+						Configs:                []string{"configmap:my-cm"},
+						Resources:              []string{"secret:my-secret"},
+						PersistentVolumeClaims: []string{"my-pvc:/over/the/rainbow"},
+						EmptyDirs:              []string{"my-empty-dir:/some/path"},
```

Comment:
We should create a new test with empty dirs instead. So we test both the default (void) and the presence of the parameter.